apps/evm/generated/VAnchor/VAnchor.ts

```diff
@@ -0,0 +1,2162 @@
+// THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+
+import {
```

Comment:
Since this file is autogenerated I dont think we should keep it in the repo, rather we should generate on build, no?